src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import App from './components/App.jsx'
+
+ReactDOM.render((
+   <Router>
+      <Route path = "/" component = {App}></Route>
+   </Router>
```

Comment:
you don't put routes in the index 
you only need to render App and routes should be in the App component